client/lib/screen-title/index.js

```diff
@@ -13,16 +14,21 @@ import { setTitle as legacySetTitle } from './actions';
 export function subscribeToStore( store ) {
 	let title = store.getState().page.title;
```

Comment:
>@aduth If you set the `const state = store.getState()` on this line, it will not receive any updates while in the `store.subscribe` callback.

Right, this makes sense. We can't reuse the same reference because the state is not mutated directly. The usefulness with assigning state to a variable is how it's used at that point in time on the subsequent lines, but not at a point in the future (e.g. after a `subscribe` change occurs).

Example:

```jsx
const state = store.getState();
let { title, count } = state.page;
let siteId = getSelectedSiteId( state );

store.subscribe( () => {
	const nextState = store.getState();
	const { title: nextTitle, count: nextCount } = nextState.page;
	const nextSiteId = getSelectedSiteId( nextState );

	// ...
} );
```